index.html

```diff
@@ -2,15 +2,19 @@
 <html>
   <head>
     <!-- Google Tag Manager -->
-    <!--This script runs Google Tag Manager on my website</script-->
-    // put in a comment...
-    <script>var root_url = "http://www.davidsguessinggame.com/"</script>
-    <title>David's Stadium Guessing Game!</title>
+    <!--This fake script runs Google Tag Manager on my website</script-->
+    <!--var root_url = "http://www.davidsguessinggame.com/"</script>-->
     <meta charset='utf-8'>
+    <meta http-equiv='X-UA-Compatible' content='IE=edge'>
+    <title>David's Stadium Guessing Game!</title>
+    <link rel='stylesheet' href="">
   </head>
 
-  <body background="made_up_tiger_stadium_image.jpg">
+  <body>
     <h1><b>David's Stadium Guessing Game!</b></h1>
+    <p></p>
```

Comment:
Observations, knowing that you may have already corrected these things:
1) If you're not using a tag, don't include it in your pull request. I see tags such as Link, P, H3, UL, etc...
2) You should know this now: Don't include style tags such as B, I, etc in your HTML. Those belong in your CSS.